make.paws/inst/extdata/packages.yml

```diff
@@ -318,6 +347,7 @@
     - cognito-identity
     - cognito-idp
     - cognito-sync
+    - codeguru-security
```

Comment:
I think this should go in developer tools with the other code guru services, since it is for analyzing code and not infra.